pkg/cmd/server/origin/legacy.go

```diff
@@ -198,6 +199,7 @@ func LegacyStorage(storage map[schema.GroupVersion]map[string]rest.Storage) map[
 					restStorage := s.(*buildconfigetcd.REST)
 					store := *restStorage.Store
 					store.DeleteStrategy = orphanByDefault(store.DeleteStrategy)
```

Comment:
This ought to actually be updated to just set it properly in the legacy strategy, right?  Followup is ok.